terraform/gcp/modules/sigstore/sigstore.tf

```diff
@@ -339,7 +339,7 @@ module "ctlog_shards" {
   cluster_name = var.cluster_name
   // NB: These are commented out so that we pick up the defaults
```

Comment:
nit: Can you move the comment below this line?

Reply:
done!